automation/TaskManager.py

```diff
@@ -425,13 +417,13 @@ def _issue_command(self, browser: Browser,
                 command_status = 'timeout'
                 self.logger.info(
                     "BROWSER %i: Timeout while executing command, %s, killing "
-                    "browser manager" % (browser.crawl_id, command[0]))
+                    "browser manager" % (browser.crawl_id, repr(command)))
 
             self.sock.send(("crawl_history", {
                 "crawl_id": browser.crawl_id,
                 "visit_id": browser.curr_visit_id,
-                "command": command[0],
-                "arguments": str(command_arguments),
+                "command": type(command),
+                "arguments": command_arguments,
```

Comment:
@englehardt and I discussed what format would be the easiest to analyze and we came to the assumption, that JSON might just be the better choice than the `repr`
```suggestion
                "arguments": json.dumps(command.__dict__).encode('utf-8),
```